test/cluster-py/multi.test.py

```diff
@@ -0,0 +1,116 @@
+import sys
+import os
+from threading import Thread
+import time
+import yaml
+from lib.tarantool_server import TarantoolServer
+from tarantool import MeshConnection
+from tarantool.const import (
+    SOCKET_TIMEOUT,
+    RECONNECT_DELAY,
+)
+from tarantool.error import NetworkError
+from tarantool.utils import ENCODING_DEFAULT
+
+REPLICA_N = 3
+ROW_N = REPLICA_N * 20
+
+def check_cluster(addrs):
```

Comment:
I think the test can be written in much easier manner. You don't need to create cluster and setup the replication, just start two instances with a bit different data. Threads and sleeps don't necessary too, the test can be written in sequencial manner (insert data to servers1, inerst data to server2, create MeshConnection, make select/call, down instance 1, make select/call, up instance 1, make select/call, down instance 2, make select/call, down instance 1, make select/call and receive an error).